go_modules

```diff
@@ -156,15 +176,16 @@ def cmd_go_mod(cmd, dir):
 def main():
     log.info(f"Running OBS Source Service: {app_name}")
 
-    log.info(f"Extracting {archive} to {outdir}")
-    extract(archive, outdir)
+    tempdir = tempfile.mkdtemp(dir=outdir, prefix='extract-')
```

Comment:
Maybe `with tempfile.TemporaryDirectory() as tempdir:` is more helpfull here. I will create it in the correct place and will not change "outdir". Also will take care of removing the directory.